cli/testflinger_cli/__init__.py

```diff
@@ -351,6 +415,32 @@ def submit_job_data(self, data):
             ) from exc
         return job_id
 
+    def submit_job_attachments(self, job_id: str, path: Path):
+        """Submit attachments archive for a job to the server
+
+        :param job_id:
+            ID for the test job
+        :param path:
+            The path to the attachment archive
+        """
+        try:
+            self.client.post_attachment(job_id, path)
+        except client.HTTPError as exc:
+            if exc.status == 400:
+                raise SystemExit(
+                    f"Unable to submit attachment archive for {job_id}"
+                ) from exc
+            if exc.status == 404:
+                raise SystemExit(
+                    "Received 404 error from server. Are you "
+                    "sure this is a testflinger server?"
```

Comment:
This can probably also happen if we're running against an old Testflinger server that doesn't support attachments yet. Not sure if we want to reflect this in the error message?

```suggestion
                    "Received 404 error from server. Are you "
                    "sure this is a testflinger server with attachments support?"
```